tests/manager/integration/license/overdrive/test_api.py

```diff
@@ -1726,166 +1730,9 @@ def test_update_licensepool_provides_bibliographic_coverage(
         assert not pool.work
         http.queue_response(200, content=availability)
         http.queue_response(200, content=bibliographic)
-        pool, was_new, changed = overdrive_api_fixture.api.update_licensepool(
-            identifier.identifier
-        )
-        assert was_new is False
-        assert pool.work.presentation_ready is True
-
-    def test_update_new_licensepool(
```

Comment:
The function that these tests target is not being used anywhere in the application code so I removed them.